third_party/mkl_dnn/mkldnn_v1.BUILD

```diff
@@ -186,6 +201,7 @@ cc_library(
             "src/cpu/aarch64/**",
             "src/cpu/rv64/**",
             "src/cpu/x64/gemm/**/*_kern_autogen.cpp",
+            "src/cpu/sycl/**",
```

Comment:
Do we need SYCL for CPU? Why are we building it by default?

Reply:
Actually, this is part of exclude = [ part and not in the src = [

Reply:
@penpornk , can you review this PR again?